pep-0690.rst

```diff
@@ -479,11 +532,25 @@ enabled.
 C API
 -----
 
-For authors of C extension modules, the proposed
-``importlib.set_lazy_imports()`` function will also be exposed in the stable C
-API as ``PyImport_SetLazyImports(PyObject *names_or_callback_or_null)``, and
-``importlib.is_lazy_import`` will be available as ``PyDict_IsLazyImport(PyObject
-*dict, PyObject *key)``.
+For authors of C extension modules, the proposed public C API is as follows:
+
+* ``importlib.set_lazy_imports(enabled: bool = True, excluding: typing.Container[str] | None = None)`` is available as
+  ``PyObject *PyImport_SetLazyImports(PyObject *enabled, PyObject *excluding)``.
+
+* ``importlib.is_lazy_import(dict: typing.Dict[str, object], name: str) -> bool`` is available as
+  ``int PyDict_IsLazyImport(PyObject *dict, PyObject *name)``.
+
+* ``importlib.is_lazy_imports_enabled() -> bool`` is available as
+  ``int PyImport_IsLazyImportsEnabled()``.
+
+* ``void PyDict_ResolveLazyImports(PyObject *dict)`` resolves all lazy objects in a
+  dictionary, if any. To be used prior calling ``PyDict_NextWithError()`` or
```

Comment:
I don't think this sentence "to be used prior calling..." makes sense -- `PyDict_Next` now does this internally, you don't have to do it first.

Reply:
`PyDict_Next()` only does it for us if we're starting in position 0, otherwise it doesn't. The safest would be to always resolve the dictionary before calling; if it has already been resolved, the resolving function does nothing anyway.

Reply:
If you want to clarify those details, that's fine, but then I would give the full context. The text that is currently there implies that you have to call it yourself without giving enough information to really understand whether/when you do or don't have to.